modules/rest_api/controllers/services/rest.php

```diff
@@ -1593,16 +1593,21 @@ private function esGetSpecialFieldHigherGeography(array $doc, array $params) {
    * @return string
    *   Formatted value.
    */
-  private function esGetSpecialFieldLat(array $doc) {
+  private function esGetSpecialFieldLat(array $doc, array $params) {
     // Check in case fields are in composite agg key.
     $root = isset($doc['key']) ? $doc['key'] : $doc['location'];
     if (empty($root['point'])) {
       return 'n/a';
     }
     $coords = explode(',', $root['point']);
-    $ns = $coords[0] >= 0 ? 'N' : 'S';
-    $lat = number_format(abs($coords[0]), 3);
-    return "$lat$ns $lon$ew";
+    switch($params[0]) {
```

Comment:
Should we allow for a default when no options provided (nssuffix?)

Reply:
Okay will do.